components/Send.tsx

```diff
@@ -9,22 +9,12 @@ import AddMultiple from "../assets/svgs/AddMultiple";
 import CustomButton from "./CustomButton";
 import CryptoDropdown from "./CryptoDrown";
 import Polygon from "../assets/svgs/Polygon";
-import { DefaultState, ModalType } from "../sdk/state";
+import { useGlobalState } from "../sdk/state";
+import { ModalScreen } from "../types/enums";
 
-interface SendProps {
-  onPress: (
-    {
-      type,
-      value,
-    }: {
-      type: keyof DefaultState;
-      value: ModalType;
-    },
-    top: number | undefined
-  ) => void;
-}
 
-export default function Send({ onPress }: SendProps) {
+export default function Send({ onPress }: any) {
```

Comment:
Can we type the `onPress` properly instead of using `any`?

Reply:
thanks